docs/modules/vector-search/pages/vector-collections.adoc

```diff
@@ -0,0 +1,498 @@
+= Vector Collection
```

Comment:
I think we should put this under [Data structures](https://docs.hazelcast.com/hazelcast/5.4/data-structures/) - it fits there best, unless we want to promote it on top level

Reply:
97e456281191372558f94773bdb743a9c1f92e4f